src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/BlockHeaderPosContextualRuleTest.cs

```diff
@@ -155,10 +158,10 @@ public BlockHeaderPosContextualRuleTest()
                 },
             };
             this.network.Consensus.LastPOWBlock = 12500;
-            this.ruleContext.BlockValidationContext.Block.Transactions[1].Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + 15;
-            this.ruleContext.BlockValidationContext.ChainedBlock.Header.Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + 15;
+            this.ruleContext.BlockValidationContext.Block.Transactions[1].Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + MaxFutureDriftAfterHardFork;
+            this.ruleContext.BlockValidationContext.ChainedBlock.Header.Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + MaxFutureDriftAfterHardFork;
```

Comment:
this is the one that still passes even if you don't make the two times relevant to the maxfuturedriftafterhardfork right?  The significance of this test seems to be that the two values are different, not that they are in any way related to being inside or outside the drift value

Reply:
Not exactly. This test tries to test https://github.com/Neurosploit/StratisBitcoinFullNode/blob/master/src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BlockHeaderPosContextualRule.cs#L68 the second part of this like after &&

Reply:
With the two times the same.... yes - I just realised I meant this comment against line 130 - where the two times are different hence it always shortcutting and never performing the second part of the &&.